### DISCUSSION TITLE
What is the cleanest way to import helper functions into a Nitro serverless function?

### DISCUSSION THREAD

**Discussion by jetlej** [nitrojs/nitro#1413] [General]
I have a group of functions that I want to import and use across many different serverless functions. 

What is the best way to import those functions?

I tried using `await useStorage().getItem()`, but that just brings in the .js file as text and I'd have to run eval() to make it actually run. Seems sloppy :/

**Comment by 1bye** [comment]
Also need this.
I have tried use:
```js
const files = await glob('./*.ts');
const instances = await Promise.all(files.map(async file => {
   const { default: ClassConstructor } = await import('./'+file);
   return new ClassConstructor();
}));
``` 
but this don't work

**Comment by cosbgn** [comment]
Anything you put inside `/utils` folder is auto imported everywhere!
